test/mailers/user_mailer_test.rb

```diff
@@ -0,0 +1,12 @@
+require 'test_helper'
+
+class UserMailerTest < ActionMailer::TestCase
+  test "password_changed" do
+    mail = UserMailer.password_changed(id)
```

Comment:
Without looking, I'm guessing the test is choking because `id` doesn't refer to anything. I'd suggest creating a user object (something like `user = create :user`) and then passing `user into the `password_changed` method. 

You're pretty much nailing the rest of the test!